pkgs/applications/networking/browsers/firefox/rdd-sandbox-paths.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/security/sandbox/linux/broker/SandboxBrokerPolicyFactory.cpp b/security/sandbox/linux/broker/SandboxBrokerPolicyFactory.cpp
+index 3b3409150772..47142889cd2c 100644
+--- a/security/sandbox/linux/broker/SandboxBrokerPolicyFactory.cpp
++++ b/security/sandbox/linux/broker/SandboxBrokerPolicyFactory.cpp
+@@ -793,6 +793,8 @@ SandboxBrokerPolicyFactory::GetRDDPolicy(int aPid) {
+   policy->AddDir(rdonly, "/usr/lib");
+   policy->AddDir(rdonly, "/usr/lib32");
+   policy->AddDir(rdonly, "/usr/lib64");
++  policy->AddDir(rdonly, "/run/opengl-driver/lib");
++  policy->AddDir(rdonly, "/nix/store");
```

Comment:
I'm wondering if this somehow breaks some security assumptions/weakens this security mechanism. Then again, I don't have any better idea for fixing this.

TBH, I know barely anything about the Firefox codebase and while I think this is probably fine, I'd personally defer judgement to upstream.